core/src/main/java/org/elasticsearch/search/suggest/SuggestUtils.java

```diff
@@ -171,12 +173,12 @@ public static SuggestMode resolveSuggestMode(String suggestMode) {
         }
     }
 
-    public static Suggest.Suggestion.Sort resolveSort(String sortVal) {
+    public static TermSuggestionBuilder.SortBy resolveSort(String sortVal) {
```

Comment:
I'd prefere this helpers in the enums themselves, so we can unclutter SuggestUtils a bit more.